.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+body:
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Environment
+      description: Where are you using our software?
+      options:
+        - SaaS (https://sentry.io/)
+        - single-tenant (if you know, you know)
```

Comment:
I'd double check this with support/sales/someone 🙂

Reply:
Meh, will remove for now to expedite, it's not on the existing form.

Reply:
2ccdba028a7b9e7741ba0bae5f7a15f2b50e7f93